src/main/java/com/malinatran/response/Response.java

```diff
@@ -1,20 +1,24 @@
 package com.malinatran.response;
 
 import com.malinatran.constants.Status;
+import com.malinatran.router.Logger;
 
 import java.util.HashMap;
 import java.util.Map;
 
 public class Response {
 
+    private static final String LOCATION = "Location";
+
     private String method;
```

Comment:
Is everything being used here? Taking a look at the entire file and I notice that `method` , `path`, and `body` aren't being used... Why is that?